src/routes/(page)/utility-classes/links/+page.md

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+    import IconMenu from "$lib/icons/IconMenu.svelte";
+    import IconAdd from "$lib/icons/IconAdd.svelte";
+</script>
+
+# Links
+
+All styles that can be applied to anchor tags.
+
+## Showcase
+
+### Default
+
+<div class="container" data-tid="showcase">
+
+| Styles                                                                            |
+| --------------------------------------------------------------------------------- |
+| <a href="#">Default Link</a>                                                      |
+| <button class="primary">Primary Button</button>                                   |
```

Comment:
Why is there a button here?

Reply:
I added it for users to be able to check that it's indeed the same style. But true that it's not necessary...